db/seeds.rb

```diff
@@ -91,7 +107,7 @@
     user = User.all.sample
     group = Group.all.sample
     if group.gm != user && !group.users.include?(user) && group.users.count < 5
-      p_group = PlayerGroup.create(
+      p_group = PlayerGroup.create!(
```

Comment:
Am thinking about putting this at the bottom of the seed and generate a Group that has users with the same preference in genre

Reply:
you could do something like
```
Genre.all.each do |genre|
  # users = players where genre: genre
  # groups = group where rulebook.genre: genre
  # assign players to these groups
end
```
alternatively you could first randomly assign players to groups, and then assign them preferences to match the groups they joined